k8s/helm-chart/dynatrace-gcp-function/templates/template.yml

```diff
@@ -77,11 +77,13 @@ spec:
             configMapKeyRef:
               name: dynatrace-gcp-function-config
               key: SERVICE_USAGE_BOOKING
+
         - name: USE_PROXY
           valueFrom:
             configMapKeyRef:
               name: dynatrace-gcp-function-config
               key: USE_PROXY
+        {{- if or (eq .Values.useProxy "ALL")  (eq .Values.useProxy "DT_ONLY") (eq .Values.useProxy "GCP_ONLY")}}
```

Comment:
expression can be simplified to `if .Values.useProxy` - empty value of useProxy means - no proxy

Reply:
Simplified